src/sources/forus-webshop/pug/tpl/pages/vouchers.pug

```diff
@@ -27,7 +27,7 @@ main(id="main-content")
                         text="{{ 'vouchers.empty_block.cta' | translate }}"
                         hide-link="false")
 
-                    .profile-content-header.clearfix(ng-if="$ctrl.productVouchers.length > 0")
+                    //-.profile-content-header.clearfix(ng-if="$ctrl.productVouchers.length > 0")
```

Comment:
title of 'reserveringen' is not necassary anymore; as there is now a new page called 'reserveringen'